frontend/src/services/UserService.ts

```diff
@@ -1,9 +1,24 @@
 import $api from '../http';
 import { AxiosResponse } from 'axios';
 import { IUser } from '../types/IUser';
+import { FieldNames } from '../pages/Account';
+import { IRequestUserData } from '../types/request/IRequestUserData';
 
 export class UserService {
   static async getUser(): Promise<AxiosResponse<IUser>> {
     return $api.get<IUser>(`/user`)
   }
+  static async updateUserFieldByField(fieldToChange: string, userData: Partial<IRequestUserData> ): Promise<AxiosResponse<IUser>> {
+    const endPointEmail: string = '/user/email';
+    const endPointPassword: string = '/user/password';
```

Comment:
Don't create separate variables for the endpoints, this is different from the current code.